### PR TITLE
addon listing page

### PR DESCRIPTION
**Pull request by spasovski**
The existing tiles on mobile will need some tweaking for zebra-striping and updated padding. That will be another PR and this page is neatly independent for now.

### CODE REVIEW

**Comment by sherlockedci** [comment]
[ The Adventure of the Norwood Build](http://sherlocked.dev.mozaws.net/builds/83968549)

File: src/media/css/addon-tiles.styl [hunk 1]
```diff
@@ -42,3 +42,45 @@
...
+// TODO: Use this for apps and move it in its own file.
+.container-list {
+    margin: 10px 0 0;
+}
+
+.addon-tile {
+    display: block;
+}
+
+a.addon-tile:hover {
```
Comment (chuckharmston) [inline src/media/css/addon-tiles.styl]: Why do you need the element selector here?

**Comment by chuckharmston** [comment]
r+

File: src/media/css/addon-tiles.styl (hunk 1 above)
Comment (spasovski) [inline src/media/css/addon-tiles.styl]: on the details page it's a `div` - everywhere else `a`

**Pull request merged**